app.js

```diff
@@ -87,12 +82,24 @@ app.once('ready', () => {
         // ipc.server.broadcast('screenCastWindow_shown', { show: true });
 
       const autoCloseScript = `
-
+        window.setInterval(function() {
+          if (document.getElementsByClassName('WEB_PAGE_TYPE_ACCOUNT_SELECTOR').length >= 1) {
+            console.log("mmm-screencast.exited");
+          }
+        }, 1000);
```

Comment:
So I do not think the `WEB_PAGE_TYPE_ACCOUNT_SELECTOR` element is constantly there (I have not been able to see it, a better solution would be to do something like this:

```suggestion
        let videoEleStop;

       // consistently check the DOM for the video element
        const interval = setInterval(() => {
          videoEleStop = document.getElementsByTagName('video')[0];

         // if the video element exists add an event listener to it and stop the interval
          if (videoEleStop) {
            videoEleStop.addEventListener('ended', (event) => {
              console.log("mmm-screencast.exited");
            });
            clearInterval(interval);
          }
        }, 1000);
```

Reply:
I merged your suggestion. :+1: